NEMO/utilities.py

```diff
@@ -74,13 +92,36 @@ def convert_py_format_to_js(string_format: str) -> str:
     return string_format
 
 
+def convert_py_to_pick_time(string_format: str) -> str:
+    try:
+        return settings.PICK_TIME_FORMAT
+    except AttributeError:
+        string_format = (
+            string_format.replace("%w", "")
+            .replace("%s", "")
+            .replace("%f", "")
+            .replace("%z", "")
+            .replace("%Z", "")
+            .replace("%j", "")
+            .replace(":%S", "")
+            .replace("%S", "")
+            .replace("%U", "")
+            .replace("%W", "")
+            .replace("%c", "")
+        )
```

Comment:
might want to add %G, %u, %V, :%z https://docs.python.org/3/library/datetime.html#format-codes